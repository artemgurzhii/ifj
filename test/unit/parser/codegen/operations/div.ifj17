n=5/6
n=5/5
n=4/3
