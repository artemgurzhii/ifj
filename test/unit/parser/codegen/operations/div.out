.IFJcode17
DIV n 5 6
DIV n 5 5
DIV n 4 3
